### DISCUSSION TITLE
Empty Table Of Contents (TOC) With 'Default 2022' Presentation style

### DISCUSSION THREAD

**Discussion by CoderWRr** [EWSoftware/SHFB#990] [Q&A] [answered]
I have a clean solution with two projects In Visual Studio 2022 (see the attachment).
1. Clean C# .net Console Application project. Just added some summary comments.
2. Clean SHFB Project with first project specified as  Documentation Sources. No other settings changed.
The 'Default 2022' Presentation Style gives html output with empty Table Of Contents (TOC) left column. While the 'VS 2013' Presentation Style gives correct output.
Could you, please, explain how to fix the 'Default 2022' Presentation Style?
Thank you!

[TestSolution.zip](https://github.com/EWSoftware/SHFB/files/11524028/TestSolution.zip)

**Comment by EWSoftware** [comment, marked as answer]
Are you viewing it by opening the content on the file system directly?  If so, the script is being blocked by the browser.  Use the option in the standalone GUI or the Visual Studio package to view the help output so that it runs through a web server instance.

**Reply by CoderWRr** [reply]
The true, original Magic!
That’s exactly what I’ve been searching for several days. Had no idea the issue was related to browsers (they gave no clue about blocking local html files to access local filesystem).
Thank You very much!
